api/v01/vrp.rb

```diff
@@ -112,15 +112,14 @@ class Vrp < APIBase
               ret = OptimizerWrapper.wrapper_vrp(api_key, APIBase.profile(api_key), vrp, checksum)
               count_incr :optimize, transactions: vrp.transactions
               if ret.is_a?(String)
-                # present result, with: VrpResult
                 status 201
-                present({ job: { id: ret, status: :queued }}, with: Grape::Presenters::Presenter)
+                present({ job: { id: ret, status: :queued }}, with: VrpResult)
               elsif ret.is_a?(Hash)
                 status 200
                 if vrp.restitution_csv
                   present(OptimizerWrapper.build_csv(ret.deep_stringify_keys), type: CSV)
                 else
-                  present({ solutions: [ret], job: { status: :completed }}, with: Grape::Presenters::Presenter)
+                  present({ solutions: [ret&.deep_symbolize_keys], job: { status: :completed }}, with: VrpResult)
```

Comment:
I belive this was your first question @fonsecadeline I think it is better if we fix this issue from the source so that the result hash always uses symbols in the first place instead of `deep_symbolize_keys`. This hash is one of the last hashes that is in the old style and it is just annoying to swithc to symbol/string in the middle of the code.